README.md

```diff
@@ -1,26 +1,266 @@
+  - PacketRxUnixTime: PC clock-driven time when the packet was received. Highly inaccurate after packet drops.
+  - packetsizes: Number of samples per packet. Written in rows corresponding to the last sample of each packet.
+  - dataTypeSequence: 8-bit packet number counter that rolls over, ranging from 0 to 255; can be used to help identify if packets are in order or are missing. Should run continuously, but instances of resetting have been observed.
+  - DerivedTime: Computed time for each sample - see *Calculating DerivedTime* for more information
+  
+- **RawDataAccel.json** --> AccelData
+  - XSamples: X-axis
+  - YSamples: Y-axis
+  - ZSamples: Z-axis
+  - timestamp
+  - samplerate
+  - PacketGenTime
+  - PacketRxUnixTime
+  - packetsizes
+  - dataTypeSequence
+  - DerivedTime
+
+- **DeviceSettings.json** 
+[examples from different files]
+  - timeDomainSettings
+  
+    ![timeDomainSettings](https://github.com/openmind-consortium/Analysis-rcs-data/blob/DocumentationUpdate/documentationFigures/timeDomain.PNG)
+  
+  - powerSettings
+  
+    ![powerSettings](https://github.com/openmind-consortium/Analysis-rcs-data/blob/DocumentationUpdate/documentationFigures/powerSettings_2.PNG)
+  
+  - fftSettings
+    ![fftSettings](https://github.com/openmind-consortium/Analysis-rcs-data/blob/DocumentationUpdate/documentationFigures/fftSettings.PNG)
+  
+  - metaData
+
+- **RawDataFFT.json** --> FFTData
+  - PacketGenTime
+  - PacketRxUnixTime
+  - Channel
+  - FftSize
+  - FftOutput
+  - Units
+  - TDsamplerate
+  - dataSize
+  - dataType
+  - dataTypeSequence
+  - globalSequence
+  - info
+  - systemTick
+  - user1
+  - user2
+  - timestamp
+  - samplerate
+  - packetsizes
+  - DerivedTIme
+    
+- **RawDataPower.json** --> PowerData
+  - dataSize
+  - dataType
+  - dataTypeSequence
+  - globalSequence
+  - info
+  - systemTick
+  - timestamp
+  - TDsamplerate
+  - PacketGenTime
+  - PacketRxUnixTime
+  - ExternalValueMask
+  - FFTSize
+  - IsPowerChannelOverrange
+  - Band1
+  - Band2
+  - Band3
+  - Band4
+  - Band5
+  - Band6
+  - Band7
+  - Band8
+  - samplerate
+  - packetsizes
+  - DerivedTime
+
+- **AdaptiveLog.json**: TBD
+- **StimLog.json**: TBD
+- **ErrorLog.json**: Not currently used
+- **EventLog.json**: TBD
+- **DiagnosticsLog.json**: TBD
+- **TimeSync.json**: Not currently used
+
+## Structure of Repository:
+- **code**
+  - functions: code for specific needs; [TBD if these are further organized in subfolders]
+  - toolboxes: turtle_son, etc...
+- **testDataSets**: benchtop generated test data sets for validation of code; often generated signals are simultaneously recorded with DAQ to allow for verification of timing across data streams. 
+- **outputFigs**: will contain the output of function testing to the specified test dataset
+
+## Functions: 
+This list contains the functions that have been tested in brach and pushed to master (brief description of function input output next to each function name)
```

Comment:
branch is misspelled as brach

Reply:
Fixed with f66235b663ee7f630ad3e8aab9fda7a1ca4f639a